chevah/geck
# Remove Trac from our documentation.

Scope
=====

As we have migrated to GitHub, we no longer need to use Trac


Changes
=======

Convert any Trac documentation into GitHub Issues

As a drive-by I have updated the site to latest python 3.8 and sphinx version


How to try and test the changes
===============================

reviewers: @danuker 

Check that changes make sense
